docs/readmes/faq/magma_faq.md

```diff
@@ -65,6 +65,22 @@ This section lists some of the commonly asked questions related to Magma operati
     - From the popup window, select tab **COMMANDS**. Then, check section **Reboot**.
     - Click on **Reboot** to reboot the AGW node.
 
+### Does AGW supports IPv6?
+  - IPv6 and dual stack IPv4V6 session support are currently under active development and we are working towards making this feature ready as part of release 1.4.
+
+### How to change hardware UUID of AGW?
+  - Hardware UUID is in `/etc/snowflake`.
+  - Delete file `/etc/snowflake` and restart magma@magmad systemd service, which will create new one.
+  - **NOTE** If you reinstall magma, the UUID hardware ID will change.
+
+### How to switch between info/debug mode in mme logs?
+  - Modify `/var/opt/magma/configs/mme.yml` and add the line `"log_level: DEBUG"`,
+  - Restart the mme service with `sudo service magma@mme restart`.
+  - **NOTE** Expect impact of service while doing above procedure.
```

Comment:
should we mention that this can be done through orc8r too?

Reply:
Mostly they try with AGW and operationally AGW is mostly active, so let's keep it now from AGW.